src/plugins/lvm-dbus.c

```diff
@@ -909,6 +916,7 @@ static BDLVMPVdata* get_pv_data_from_props (GVariant *props, GError **error) {
     }
 
     vg_props = get_object_properties (value, VG_INTF, error);
+    g_free (value);
```

Comment:
Shouldn't the above `if` block free `value` too?

Reply:
Correct.

So just a hint here - `GVariant` format strings allow either handing ownership or keeping ownership of the memory. In this case changing the format string to `&o` and the type of the variable to `const gchar *` would save you the need for explicit deallocation. Just be sure not to free the enclosing `GVariant` while this string is still used (it's stack allocated here though).

Reply:
Changing to `&o`, thanks for the tip!